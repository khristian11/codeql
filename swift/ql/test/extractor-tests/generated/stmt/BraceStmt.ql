import codeql.swift.elements
import TestUtils

from BraceStmt x, string isUnknown
where
  toBeTested(x) and
  if x.isUnknown() then isUnknown = "isUnknown" else isUnknown = "not isUnknown"
select x, isUnknown
