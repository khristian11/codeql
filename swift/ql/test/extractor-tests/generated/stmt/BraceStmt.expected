| statements.swift:1:13:32:1 | { ... } | not isUnknown |
| statements.swift:2:26:8:3 | { ... } | not isUnknown |
| statements.swift:3:15:5:5 | { ... } | not isUnknown |
| statements.swift:5:12:7:5 | { ... } | not isUnknown |
| statements.swift:10:26:12:3 | { ... } | not isUnknown |
| statements.swift:15:18:17:3 | { ... } | not isUnknown |
| statements.swift:19:6:21:3 | { ... } | not isUnknown |
| statements.swift:21:11:23:3 | { ... } | not isUnknown |
| statements.swift:25:6:27:3 | { ... } | not isUnknown |
| statements.swift:27:26:29:3 | { ... } | not isUnknown |
| statements.swift:29:11:31:3 | { ... } | not isUnknown |
| statements.swift:38:31:42:1 | { ... } | not isUnknown |
| statements.swift:39:21:41:3 | { ... } | not isUnknown |
| statements.swift:44:1:46:1 | { ... } | not isUnknown |
| statements.swift:44:7:46:1 | { ... } | not isUnknown |
| statements.swift:48:1:50:1 | { ... } | not isUnknown |
| statements.swift:48:4:50:1 | { ... } | not isUnknown |
| statements.swift:52:1:52:13 | { ... } | not isUnknown |
| statements.swift:53:1:62:1 | { ... } | not isUnknown |
| statements.swift:55:7:56:7 | { ... } | not isUnknown |
| statements.swift:58:7:59:7 | { ... } | not isUnknown |
| statements.swift:61:7:61:22 | { ... } | not isUnknown |
| statements.swift:64:1:64:15 | { ... } | not isUnknown |
| statements.swift:65:1:66:1 | { ... } | not isUnknown |
| statements.swift:65:21:66:1 | { ... } | not isUnknown |
| statements.swift:67:1:68:1 | { ... } | not isUnknown |
| statements.swift:67:22:68:1 | { ... } | not isUnknown |
| statements.swift:70:1:70:23 | { ... } | not isUnknown |
| statements.swift:71:1:72:1 | { ... } | not isUnknown |
| statements.swift:71:45:72:1 | { ... } | not isUnknown |
| statements.swift:75:7:75:7 | { ... } | not isUnknown |
| statements.swift:75:7:75:7 | { ... } | not isUnknown |
| statements.swift:75:7:75:7 | { ... } | not isUnknown |
| statements.swift:76:7:76:7 | { ... } | not isUnknown |
| statements.swift:77:13:79:5 | { ... } | not isUnknown |
| statements.swift:81:9:83:5 | { ... } | not isUnknown |
